execution/executor-benchmark/src/main.rs

```diff
@@ -107,6 +108,12 @@ pub struct PipelineOpt {
     #[clap(long, default_value = "1")]
     num_executor_shards: usize,
     #[clap(long)]
+    remote_sharding: bool,
```

Comment:
Can you take a stab at creating a new ShardingOpt struct and move them there? Also, we don't need a separate `remote_sharding` flag, it should be enabled when `remote_executor_addresses` is set ?

Reply:
Done